modules/envivoBidAdapter.js

```diff
@@ -0,0 +1,129 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import * as utils from '../src/utils';
+import {BANNER, VIDEO} from '../src/mediaTypes';
+import { ajax } from '../src/ajax';
+import {Renderer} from '../src/Renderer';
+
+const SUPPORTED_AD_TYPES = [BANNER, VIDEO];
+const BIDDER_CODE = 'envivo';
+const DOMAIN = 'http://ad.nvivo.tv/';
```

Comment:
We're deprecating the use of `http` with the `prebid-3.0` release. We recommend you change this to `https` to be part of the new **prebid-3.0** release.

Reply:
As you said, we have changed from http to https.